lib/internal/event_target.js

```diff
@@ -117,6 +117,10 @@ class Event {
   }
 
   get [Symbol.toStringTag]() { return 'Event'; }
+  static NONE = 0;
+  static CAPTURING_PHASE = 1;
+  static AT_TARGET = 2;
+  static BUBBLING_PHASE = 3;
```

Comment:
Adding a comment in here that we currently do not use the `CAPTURING_PHASE` and `BUBBLING_PHASE` but that they need to be preserved anyway because of the spec would be good.